style.css

```diff
@@ -1,4 +1,623 @@
+  margin-top: 16px;
+  width: 33%;
+}
+
+.scroll__btn {
+  background-color: white;
+  border-radius: 15px;
+  cursor: pointer;
+  transform: translateX(10px);
+  z-index: 1;
+}
+
+.scroll__wheel {
+  background: #000;
+  border-radius: 19px;
+  height: 8px;
+  margin-bottom: 15px;
+  transform: translateX(-5px);
+  width: 3px;
+  z-index: 2;
+}
+
+@media (min-width: 1024px) and (max-width: 1280px) {
+  * {
+    box-sizing: border-box;
+  }
+
+  html,
+  body {
+    margin: 0 auto;
+    max-width: 1280px;
+    min-width: 1024px;
+    overflow-x: hidden;
+    width: 100%;
+  }
+
+  .header__logo {
+    cursor: pointer;
+    display: flex;
+    height: 92px;
+    margin-right: 10vw;
+    width: 170px;
+  }
+
+  .language {
+    cursor: pointer;
+    display: flex;
+    height: 21px;
+    margin-bottom: 0;
+    margin-top: 18px;
+    padding: 0;
+    width: 50px;
+  }
+
+  .language__img {
+    height: 21px;
+    width: 24px;
+  }
+
+  .language__arrow {
+    height: 8px;
+    margin: 7px;
+    width: 12px;
+  }
+
+  .header-menu {
+    right: 0;
+  }
+
+  .collection {
+    padding-left: 9vw;
+    padding-right: 9vw;
+    padding-top: 50px;
+  }
+
+  .collection__img {
+    margin-right: 4vw;
+    height: 45%;
+    width: 40%;
+  }
+
+  .collection__title {
+    font-size: 70px;
+    font-style: normal;
+    font-weight: bold;
+    line-height: 72px;
+    margin-top: 60px;
+  }
+
+  .collection__description {
+    margin-bottom: 2.5vw;
+  }
+
+  .collection__btn {
+    font-size: 12px;
+    margin-right: 3vw;
+  }
+
+  .range__link {
+    font-size: 12px;
+  }
+}
+
+@media (max-width: 1024px) {
+  .header__logo {
+    margin-right: 7vw;
+  }
+
+  .language {
+    display: none;
+  }
+
+  .collection {
+    padding-top: 45px;
+  }
+
+  .collection__title {
+    font-size: 42px;
+    line-height: 43px;
+    margin-bottom: 2vw;
+    margin-top: 20px;
+  }
+
+  .collection__description {
+    font-size: 13px;
+    line-height: 26px;
+    margin-bottom: 2vw;
+  }
+
+  .collection__btn {
+    font-size: 11px;
+    margin-right: 2.5vw;
+  }
+
+  .range__link {
+    color: #000;
+    font-size: 11px;
+    margin-right: 1vw;
+  }
+
+  .scroll {
+    display: none;
+  }
 }
 
+@media (max-width: 880px) {
+  .collection__img {
+  	height: 35%;
+  	width: 35%;
+  }
+
+  .collection {
+  	padding-right: 11vw;
+  }
+
+  .collection__img {
+    margin-bottom: 7vw;
+}
+
+  .collection__title {
+  	margin-top: 7vw;
+  }
+
+  .collection__description {
+    font-size: 15px;
+    line-height: 30px;
+    margin-bottom: 3vw;
+  }
+
+  .collection__btn {
+  	font-size: 12px;
+  	margin-right: 4vw;
+  }
+
+  .range__link {
+  	font-size: 12px;
+  }
+}
+
+@media (min-width: 568px) and (max-width: 767px) {
+  * {
+    box-sizing: border-box;
+  }
+
+  html,
+  body {
+    margin: 0 auto;
+    max-width: 767px;
+    min-width: 568px;
+    overflow-x: hidden;
+    width: 100%;
+  }
+
+  .main-page {
+  	background: none;
+  	padding: 0;
+  	margin: 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5658198)